tools/sign_release.sh

```diff
@@ -76,13 +76,17 @@ zip --quiet --delete $BUILD_DIR/app-amazonWebview-release-unsigned.apk \
     'META-INF/services/*' \
     'META-INF/web-fragment.xml' || exit 1
 
-# Align and sign via autograph. Signing can be found here:
+# Sign via autograph. Signing can be found here:
 # https://github.com/mozilla-services/autograph/blob/a1bee1add785ae41a284b1d5873010817d1fa79f/signer/apk/jar.go#L69-L71
 curl -F "input=@$BUILD_DIR/app-amazonWebview-release-unsigned.apk" \
-    -o $BUILD_DIR/app-amazonWebview-release.apk \
+    -o $BUILD_DIR/$FINAL_NAME \
     -H "Authorization: $AUTH_TOKEN" \
     $SERVER || exit 1
 
+# Align ZIP to 4 byte addresses
+zipalign -v 4 $BUILD_DIR/$FINAL_NAME $BUILD_DIR/app-amazonWebview-release-aligned.apk
```

Comment:
@Baron-Severin If these commands fail, we should fail the build, right? The new script doesn't do that (i.e. no `exit 1`).

Maybe we should do `set -e` at the top of the script to exit when any command fails instead of `exit 1`? I used exit 1 because I didn't know how to write shell scripts :P. Description of `set -e`:  https://stackoverflow.com/a/19622569

Reply:
Adding that in as part of https://github.com/mozilla-mobile/firefox-echo-show/pull/289